frontend/Plugin.js

```diff
@@ -0,0 +1,173 @@
+import SgTrackingPlugin from '@shopgate/tracking-core/plugins/Base';
```

Comment:
Hmmm, for me it's working fine. Maybe some github issues yesterday...let's check together